cmd/ocm/create/cluster/cmd.go

```diff
@@ -131,7 +137,7 @@ func init() {
 	fs.IntVar(
 		&args.computeNodes,
 		"compute-nodes",
-		4,
+		0,
 		"Number of worker nodes to provision per zone. Single zone clusters need at least 4 nodes, "+
```

Comment:
this should be changes as well to: Number of worker nodes to provision per zone. Single zone clusters need at least 2 nodes, multizone clusters need at least 3 nodes

Reply:
Great catch, this was inconsistent!

But is it really per-zone?  Current code is sending them as-is `cmv1.NewClusterNodes().Compute(config.ComputeNodes)`.  
I see UI forces multiples of 3 by showing "1", "2" options with "(per zone)", and also shows the computed × 3 zones, and sending the computed × 3 zones.  And the server validates `spec.Nodes.Compute` is multiple of 3.

So, until now this flag meant total number of nodes.
If you want to change that we should rename it `--compute-nodes-per-zone`.  Or keep `--compute-nodes` for compatibility but _also_ add a `--compute-nodes-per-zone` flag, and prompt for per-zone value.

Reply:
- [ ] if this says anything at all about minimums, it should explain both CCS and Red Hat infra minimums.

Reply:
Now says:
```
      --compute-nodes int               Number of worker nodes to provision. Single zone clusters need at least 4 nodes on Red Hat infra, 2 on CCS. Multi-AZ at least 9 nodes on Red Hat infra, 3 on CCS, and must be a multiple of 3.
```

Reply:
Now says:
```
      --compute-nodes int               Number of worker nodes to provision. Single zone clusters need at least 4 nodes on Red Hat infra, 2 on CCS. Multi-AZ at least 9 nodes on Red Hat infra, 3 on CCS, and must be a multiple of 3. If omitted, uses minimum.
```